felix/aws/ec2.go

```diff
@@ -52,7 +52,7 @@ func convertError(err error) string {
 	return fmt.Sprintf("%v", err.Error())
 }
 
-func retriable(err error) bool {
+func retryable(err error) bool {
```

Comment:
both spellings were present at roughly even levels, there's a well reasoned argument that this is the more appropriate spelling -- although I'm not wed to it, all I want is consistency.